backend/src/vectorstores/faiss.py

```diff
@@ -99,13 +136,41 @@ def add_md_manpages(
 
         return None
 
+    def add_html(
+        self, folder_paths: list[str], chunk_size: int = 500, return_docs: bool = False
+    ) -> Optional[list[Document]]:
+        if self.print_progress:
+            print('Process HTML docs...')
+
+        docs_processed: list[Document] = []
+        for folder_path in folder_paths:
+            if self.print_progress:
+                print(f'Processing [{folder_path}]...')
```

Comment:
is this a bug? will this not run if self.print_progress is false

Reply:
Add to issue: Migrate print statements to Python logging https://docs.python.org/3/library/logging.html

Messages like these can then be placed under DEBUG

Reply:
@palaniappan-r Have you fixed this?

Reply:
Yes, please check now.

I've made sure that all log messages use the `logging` module.